src/fairchem/data/oc/core/slab.py

```diff
@@ -87,17 +87,17 @@ def from_bulk_get_random_slab(
         else:
             # If we're not saving all slabs, just tile and tag one
             assert bulk is not None
-            untiled_slabs = compute_slabs(
-                bulk.atoms,
-                max_miller=max_miller,
+            bulk_struct = standardize_bulk(bulk.atoms)
+            avail_millers = get_symmetrically_distinct_miller_indices(
+                bulk_struct, max_miller
+            )
+            sampled_miller_idx = np.random.randint(len(avail_millers))
+            slabs = Slab.from_bulk_get_specific_millers(
+                avail_millers[sampled_miller_idx], bulk
             )
-            slab_idx = np.random.randint(len(untiled_slabs))
-            unit_slab_struct, millers, shift, top, oriented_bulk = untiled_slabs[
-                slab_idx
-            ]
-            slab_atoms = tile_and_tag_atoms(unit_slab_struct, bulk.atoms, min_ab=min_ab)
 
-            return cls(bulk, slab_atoms, millers, shift, top, oriented_bulk)
+            # If multiple shifts exist, sample one randomly
+            return np.random.choice(slabs)
```

Comment:
nit, same comment as above about using `default_rng` as the recommended interface. Though not sure if its better to make it a simple BE task to update uses of `np.random` in a separate PR.

Reply:
Yeah I'd prefer we do this in a separate PR as it's not specific to the solvent code but rather all of our `data/oc` code.